src/classes/index.ts

```diff
@@ -45,14 +52,22 @@ export class AxiosRedis {
    * @param {undefined | number | null} durationInMS
    * @returns Promise<string | void>
    */
-  async setCache(key: string, data: AxiosResponse, durationInMS: undefined | number | null): Promise<string | void> {
-    if (durationInMS === 0 || typeof durationInMS === 'object') {
+  async setCache(key: string, data: AxiosResponse, durationInMS: undefined | number | string | null): Promise<string | void> {
+    if (durationInMS === 0 || typeof durationInMS === 'object' || typeof durationInMS === null || durationInMS === '0') {
       return;
     }
 
-    const duration: number = durationInMS || this.config.expirationInMS;
+    let duration: number;
+
+    if (typeof durationInMS === 'string') {
+      duration = parseInt(durationInMS, 10);
+    }
+
+    duration = duration || this.config.expirationInMS;
+
+    const compressedData: string = await compress(flatted.stringify(data));
```

Comment:
Since this breaks compatibility in a big way, we should either make it a major version bump, or add an argument (or config flag) to enable compression when storing with a default to `false`.

Reply:
We gonna release a new major version to avoid to handle backward compatibility or handle this feature as optional.